agent/bench-scripts/postprocess/fio-postprocess

```diff
@@ -58,31 +49,31 @@ my @cpu_busy;	# each array element contains a hash with
 		# hostname: hostname or IP
 		# role: client, server, host, kvm-host, container-host, etc.
 		# timeseries: a hash of timestamp:value key-pairs
-my %efficiency;
-my %latency;    # a hash of latency-type:array-of-latency-values,
+# my %efficiency;
```

Comment:
The various `*-postprocess` scripts have at their cores a similar structure, including a series of top-level data structures.  However, not all of the scripts contain code to populate all of the data structures.  As such, I've commented out the declarations which are unused (for sound reasons...), but I've left them in place to maintain the commonality (and in case we should decide to enhance the code).

So, unless you insist, I'll decline.